fcrepo4/fcrepo4
# Update to use fcrepo-storage-ocfl

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3405

# What does this Pull Request do?

Updates Fedora to use `fcrepo-storage-ocfl`. All existing behavior should work the same as before.

# How should this be tested?

Run Fedora and execute any of the currently supported operations and everything should work the same as before.

# Interested parties

@fcrepo4/committers
